app/views/bookmarks/_bookmark_user_module.html.erb

```diff
@@ -1,11 +1,12 @@
 <% # expects "bookmark" %>
 <div class="<% if is_author_of?(bookmark) %>own <% end %>user module group">
-  <% blurb_cache_key = (is_author_of?(bookmark) ? "bookmark-owner-blurb-#{bookmark.cache_key}-v2" : "bookmark-blurb-#{bookmark.cache_key}-v2") %>
+  <!--bookmarker-->
+  <h5 class="byline heading">
+    <%= t("Bookmarked by") %> <%= link_to(bookmark.pseud.byline, user_pseud_bookmarks_path(bookmark.pseud.user, bookmark.pseud)) %>
```

Comment:
The test error is happening because you switched from `ts` to `t`. The switch is good great, `t` is preferred. But then the translation call should be something like `t(.bookmarked_by_html, pseud_link: link_to(....))` and the translation needs to be in `config/locales/views/en.yml`, in this case in a new section for bookmarks:
```yml
bookmarks:
  bookmark_user_module:
    bookmarked_by_html: Bookmarked by %{pseud_link}
```